### ISSUE TITLE
Ability to update template without workspace start

### ISSUE THREAD

**Issue by NiklasRosenstein** [coder/coder#12876]
Coder version: 2.10.0

When a template modifies it's parameters, I might want to change these parameters before the workspace starts. I can't find an option to do this now.

**Comment by sreya** [comment]
Yeah that would be nice (also not the first time I've heard this requested) cc @bpmct what do you think?

**Comment by bpmct** [comment]
Sure :) Perhaps we do this as a CLI option first

**Comment by strike** [comment]
Any updates?
